src/app/content/highlights/components/HighlightStyles.tsx

```diff
@@ -229,7 +230,12 @@ export const InfoIconWrapper = styled.span`
 `;
 
 // tslint:disable-next-line:variable-name
-export const CloseIcon = styled((props) => <Times {...props} aria-hidden='true' focusable='false' />)`
+export const CloseIconWrapper = styled(PlainButton)`
+  cursor: pointer;
```

Comment:
does plainbutton not have this already?

Reply:
lint was complaining about it being empty so I just added that to put something

Reply:
so why do you have to make a new styled component at all? you can just `export const CloseIconWrapper = PlainButton` and then you can still switch back to styled component if you decide you need something